examples/Extensions/AddAffiliateLocationExtensions.cs

```diff
@@ -0,0 +1,439 @@
+            foreach (Feed feed in feeds)
+            {
+                FeedOperation operation = new FeedOperation()
+                {
+                    Remove = feed.ResourceName,
+                };
+                operations.Add(operation);
+            }
+            FeedServiceClient feedService = client.GetService(
+                Services.V3.FeedService);
+
+            feedService.MutateFeeds(customerId.ToString(), operations.ToArray());
+        }
+
+        private static CustomerFeed[] GetLocationExtensionCustomerFeeds(GoogleAdsClient client,
+            long customerId)
+        {
+            List<CustomerFeed> customerFeeds = new List<CustomerFeed>();
+            GoogleAdsServiceClient googleAdsService = client.GetService(
+                Services.V3.GoogleAdsService);
+
+            // Create the query. A location extension customer feed can be identified by filtering
+            // for placeholder_types=LOCATION (location extension feeds) or
+            // placeholder_types =AFFILIATE_LOCATION (affiliate location extension feeds)
+            string query = $"SELECT customer_feed.resource_name, customer_feed.feed, " +
+                $"customer_feed.status, customer_feed.matching_function.function_string from " +
+                $"customer_feed " +
+                $"WHERE customer_feed.placeholder_types CONTAINS " +
+                $"ANY(LOCATION, AFFILIATE_LOCATION) and customer_feed.status=ENABLED";
+
+            PagedEnumerable<SearchGoogleAdsResponse, GoogleAdsRow> result =
+                googleAdsService.Search(customerId.ToString(), query);
+
+            foreach (GoogleAdsRow row in result)
+            {
+                customerFeeds.Add(row.CustomerFeed);
+            }
+            return customerFeeds.ToArray();
+        }
+
+        /// <summary>
+        /// Deletes the customer feeds.
+        /// </summary>
+        /// <param name="client">The Google Ads client.</param>
+        /// <param name="customerId">The customer ID for which the call is made.</param>
+        /// <param name="customerFeeds">The customer feeds to delete.</param>
+        private static void DeleteCustomerFeeds(GoogleAdsClient client, long customerId,
+            CustomerFeed[] customerFeeds)
+        {
+            List<CustomerFeedOperation> operations = new List<CustomerFeedOperation>();
+            foreach (CustomerFeed customerFeed in customerFeeds)
+            {
+                CustomerFeedOperation operation = new CustomerFeedOperation()
+                {
+                    Remove = customerFeed.ResourceName,
+                };
+                operations.Add(operation);
+            }
+
+            CustomerFeedServiceClient feedService = client.GetService(
+                Services.V3.CustomerFeedService);
+
+            feedService.MutateCustomerFeeds(customerId.ToString(), operations.ToArray());
+        }
+
+        /// <summary>
+        /// Gets the Google My Business feed mapping.
```

Comment:
"the Affiliated Location Extension feed mapping"

Reply:
Done